tests/integration/components/events/view/overview/event-tickets-test.js

```diff
@@ -10,4 +10,10 @@ module('Integration | Component | events/view/overview/event tickets', function(
     await render(hbs`{{events/view/overview/event-tickets}}`);
     assert.ok(this.element.innerHTML.trim().includes('Tickets'));
   });
+  test('Currency Helper', async function(assert) {
+    this.set('paymentCurrency', '$');
+    await render(hbs`{{currency-symbol data.event.paymentCurrency}}`);
+    assert.equal(this.element.textContent.trim(), '$');
+  });
 });
```

Comment:
@CosmicCoder96  @iamareebjamal , A little help needed, Travis says not able read the 'property symbol' , Is something wrong with the test i added or i am completely on a wrong path.

Reply:
@shreyanshdwivedi If you could see some possible error in this , Why travis is failing?

Reply:
@uds5501 nope even after removing the test it tells unknown attribute ‘symbol’